code/src/components/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import moment from 'moment'
+import { useSelector } from 'react-redux'
+
+const Header = () => {
+    const items = useSelector((store) => store.todos.items)
+    const completedItems = items.filter(items => items.isComplete)
+
+  return (
+    <>
+      <h1>To do List</h1>
+      <h3>
+       {moment().format('ddd MMMM Do YYYY')}
+      </h3>
+      <br/>
+      <div className="statistics">
+        <p>Total: {items.length}</p>
+        <p className={items.length > 0 ? "" : "completedTodos-none"}>Completed: {completedItems.length}</p>
```

Comment:
Nice that you show "Completed: " only when there are items in the list!! 👍

Another way to do this without css classes could be:
`{items.length > 0 && (<p>Completed: {completedItems.length}</p>}` This will make the `<p>` tag only shown when items.length is bigger than 0.

With that said, I don't think there's anything wrong with your way tho, I just wanted to mention it 😊